Production/crab/config/2017/QCD.txt

```diff
@@ -0,0 +1,33 @@
+Automatic
+globalTag=94X_mc2017_realistic_v14 sampleType=Fall17MC applyTriggerMatch=True saveGenTopInfo=False saveGenBosonInfo=True saveGenJetInfo=True
+
+QCD_Pt-120to170_MuEnrichedPt5_TuneCP5_13TeV_pythia8	1000	/QCD_Pt-120to170_MuEnrichedPt5_TuneCP5_13TeV_pythia8/RunIIFall17MiniAODv2-PU2017_12Apr2018_94X_mc2017_realistic_v14-v1/MINIAODSIM
```

Comment:
Could you please, put them in order by pt and split into two blocks (within the same file)? Otherwise, it is hard to spot if some sample is missing. For example, `/QCD_Pt_3200toInf_TuneCP5_13TeV_pythia8/RunIIFall17MiniAODv2-PU2017_12Apr2018_94X_mc2017_realistic_v14-v1/MINIAODSIM` is missing...